offscreen/manual.md

```diff
@@ -0,0 +1,140 @@
+<!--
+SPDX-License-Identifier: MPL-2.0
+
+This file is part of Ramses Composer
+(see https://github.com/GENIVI/ramses-composer-docs).
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+
+# Offscreen rendering example
+
+![](./docs/viewport_preview.png)
+
+This example shows how to perform offscreen rendering with the Ramses Composer (also called render-to-texture).
+This article explains the following concepts:
```

Comment:
The new line is not applied:
![image](https://user-images.githubusercontent.com/25718295/141133004-7d5c2802-4d01-4a9c-a2e2-500f3c4dd3ca.png)

Reply:
It's not supposed to. 2 lines in markdown means newline in the rendered version. This \n is just to keep the line a bit shorter

Reply:
Ok, still it's a bit crooked: This example shows  ... next sentence ... this article explains